packages/paste-theme/src/styles/global.ts

```diff
@@ -3,6 +3,27 @@ import {css} from '@twilio-paste/styling-library';
 export const pasteGlobalStyles = css({
   html: {
     fontSize: '100%',
+    fontFamily: 'fontFamilyText',
+  },
+  'html:lang(ja), html:lang(ja-JP)': {
+    '.paste-theme-provider, .paste-theme-provider input, .paste-theme-provider button, .paste-theme-provider textarea, .paste-theme-provider select, .paste-theme-provider ::placeholder': {
```

Comment:
question: Why do we need the element qualified selectors? It looks like if you set font-family to inherit on those components, it should just work, even for placeholder text? https://codepen.io/SiTaggart/pen/zYpGXMv

Reply:
Oh! If that's the case, I can remove them. I think this was included in the ticket just to be extra sure they had the right font.

Reply:
Oh! Do we want everything that is a child of the `ThemeProvider` to get the font, even if its not a Paste component? Thats the only reason I can think of to keep the selectors.

Reply:
Kinda, the way we do it now gives us a little wriggle room for migration from "old stuff". In that, we try to not set global css styles so that we don't affect random components thrown into the mix. Because we don't own them, we don't really have a right to change their styles, out side of the very minimum amount of container styles we must set on the provider.

If it doesn't work for you, you should move to Paste components to get all the theming and goodness for free.

For that reason, avoiding element selectors would be preferable.